pkg/sql/parser/sql.y

```diff
@@ -6060,11 +6060,25 @@ pause_jobs_stmt:
       Command: tree.PauseJob,
     }
   }
+| PAUSE JOB a_expr WITH REASON '=' string_or_placeholder
```

Comment:
Sorry for the late comment, but I'm curious what the motivation was to include the `=` token? Was `PAUSE JOB a_expr WITH REASON string_or_placeholder` an ambiguous syntax?